dll/cpl/timedate/dateandtime.c

```diff
@@ -45,11 +46,20 @@ SystemSetLocalTime(LPSYSTEMTIME lpSystemTime)
                 GetLastError() == ERROR_SUCCESS)
             {
                 /*
-                 * We successfully enabled it, we're permitted to change the system time
+                 * We successfully enabled it, we're permitted to change the time
```

Comment:
Adding full stops at the end of the sentences? I've had a hard time understanding this comment.

Reply:
Please feel free to edit as needed for better understanding.